### ISSUE TITLE
After upgrading from 1.5.10 index isn't working anymore in combination with loop=false

### ISSUE THREAD

**Issue by MaartenAnckaert** [leecade/react-native-swiper#566]
### Which OS ?
IOS

### Version
Which versions are you using:

- react-native-swiper v1.5.10
- react-native v0.48

### Expected behaviour
Page starts on provided index.

### Actual behaviour
Page just starts at index 0.

### How to reproduce it>
To help us, please fork this component, modify one example in examples folder to reproduce your issue and include link here.
-

### Steps to reproduce
1. Add a loop={false} and index={1} to any swiper.
2. Try it out.
3. It will start swiping from index 0.

**Comment by enahum** [comment]
yup same problem here.. also won't recalculate dimensions if the swiper changes width.

also loop will only let me go forward the first time.

**Comment by glenndebacker** [comment]
I'm also hitting this problem. According the  dot indicator  i'm on view 3 but it renders the first view.

**Comment by 007agentHP** [comment]
same here. even setting index to 0 and using scrollBy() in componentDidMount() results in the same issue

**Comment by 007agentHP** [comment]
Actually, which version were you on prior that it was working?

**Comment by nixoz** [comment]
I guess is the react-native update caused it.
Having these 0.45 fully working, switching to 0.48 caused problems.

**Comment by IjzerenHein** [comment]
This was just released, is it working for any of you?
https://github.com/leecade/react-native-swiper/releases/tag/1.5.13

**Comment by Rookiewan** [comment]
@IjzerenHein v1.5.13 has resolved my problem
